server/service/appconfig.go

```diff
@@ -166,6 +171,8 @@ func (svc *Service) ModifyAppConfig(ctx context.Context, p []byte) (*fleet.AppCo
 	}
 
 	oldSmtpSettings := appConfig.SMTPSettings
+
+	// TODO: enforce unique project key?
```

Comment:
What do you mean?

Reply:
Should we check if there is more than one Jira integration with the same Jira project key? Likewise, if there is more than one Zendesk integration for the same Zendesk group id? 
From the end user's perspective I think it would look like we are sending multiple tickets to the same destination for the same vulnerability.

Reply:
Link to [resolution](https://github.com/fleetdm/fleet/pull/5372#discussion_r861026173)